tests/outputs/u_institutions/institutions.yaml

```diff
@@ -0,0 +1,38 @@
+"columbiau":
+  "aka":
+    - "Columbia University"
+    - "Columbia"
+    - "ucolumbia"
+  "city": "New York"
+  "country": "USA"
+  "day": 30
+  "departments":
+    "physics":
+        "name": "Department of Physics"
+        "aka":
+            - "Dept. of Physics"
+            - "Physics"
+    "chemistry":
+        "name": "Department of Chemistry"
+        "aka":
+            - "Chemistry"
+            - "Dept. of Chemistry"]
+    "apam":
+        "name": "Department of Applied Physics and Applied Mathematics"
+        "aka":
+            - "APAM"
+        "month": "May"
+  "name": "Columbia University"
+  "schools":
+    "seas":
+    "name": "School of Engineering and " "Applied Science"
+    "aka":
+          - "SEAS"
+          - "Columbia Engineering"
+          - "Fu Foundation School of Engineering " "and Applied Science"
+  "state": "NY"
+  "street": "500 W 120th St"
+  "updated": "2020-05-30"
```

Comment:
what should happen to this when you update?  I think this may be right because in the tests it makes "today's" date into a fixed date so the tests can always pass, and that date may be this one, but please just check.

Reply:
I added a '--date' field and passed a date to the update!